extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -295,7 +295,12 @@ export class AmpAnalytics extends AMP.BaseElement {
     );
 
     this.transport_.maybeInitIframeTransport(
-      this.win,
+      // In the case of FIE rendering, we should be using the parent doc win.
+      // Currently, getAmpDoc returns the parent doc for FIE elements.
+      // This behavior will be changed by ampdoc-fie launch.
+      // TODO: this will be a blocker to launch ampdoc-fie.
+      // We should find a different way to get the parent win in case of FIE.
+      this.getAmpDoc().win,
```

Comment:
Can you instead use `getTopWindow(this.win)`? The `getTopWindow` is defined in the service.js.

Reply:
Shall we do in a separate PR as this PR will be cherry picked?
Or you think it's safe enough?

Reply:
Ok to move to a separate PR.